lib/taiko.js

```diff
@@ -61,7 +61,7 @@ const path = require('path');
 const { eventHandler } = require('./eventBus');
 const overlayHandler = require('./handlers/overlayHandler');
 const { highlightElement } = require('./elements/elementHelper');
-const numRetries = process.env.TAIKO_CRI_CONNECTION_RETRIES || 10;
+const numRetries = process.env.TAIKO_CRI_CONNECTION_RETRIES || 50;
```

Comment:
I.M.O it should be inside the [config](https://github.com/getgauge/taiko/blob/master/lib/config.js).